check.go

```diff
@@ -62,6 +63,11 @@ type HasRecentDeployCheckFragment struct {
 	Days int `graphql:"days"`
 }
 
+type HasDocumentationCheckFragment struct {
+	DocumentType    string `graphql:"documentType"`
```

Comment:
These should be the Enum types so they stay strongly typed when someone on the golang side using this library goes to use them.